nullaway/src/main/java/com/uber/nullaway/NullabilityUtil.java

```diff
@@ -160,6 +161,23 @@ public static TreePath findEnclosingMethodOrLambdaOrInitializer(TreePath path) {
     return Stream.concat(symbol.getAnnotationMirrors().stream(), typeUseAnnotations);
   }
 
+  /**
+   * Retrieve the string value inside an annotation without statically depending on the type.
+   *
+   * @param annotName Annotation name to retrieve it's value.
+   * @param methodSymbol A method which has an @Contract annotation.
```

Comment:
It's not specific to `@Contract` anymore; should be any annotation of type `annotName` I think